src/registrar/assets/js/get-gov.js

```diff
@@ -530,7 +530,7 @@ function hideDeletedForms() {
   let isDotgovDomain = document.querySelector(".dotgov-domain-form");
   // The Nameservers formset features 2 required and 11 optionals
   if (isNameserversForm) {
-    cloneIndex = 2;
+    // cloneIndex = 2;
```

Comment:
I think this was needed in an earlier iteration of the code. Right now it's in the way and keeps you from adding a form if you load the page with only 2 forms (like after a failed submission). Keeping it commented out until the dust clears on the PR